utils/src/main/java/org/corfudb/utils/lock/states/HasLeaseState.java

```diff
@@ -103,6 +103,7 @@ public HasLeaseState(Lock lock) {
      */
     @Override
     public void onEntry(LockState from) {
+        log.debug("Lock Transition from {} to HasLeaseState. Start lease renewal and monitor!", from.getType());
```

Comment:
are you sure that we won't pollute logs with this message?

Reply:
Yes, the transition won't happen frequently.